pyproject.toml

```
@@ -36,6 +36,7 @@ dependencies = [
     "numexpr",
     "xarray>=2024.03.0",
     "netcdf4",
+    "h5netcdf",
```

Why is `h5netcdf` added? Can it not also replace `netcdf4`? Not sure if we wanna have both as dependency and not one of them optional.

---

h5netcdf is somehow required, as the datastructure changed. It seems that the nc files within the zip folder use a different compression. There is so much going on at the CDS,, not completly clear from my side.
One comment on the forum mentions the general problem here: https://forum.ecmwf.int/t/problems-parsing-nc-files-after-outage/8060/3

For me netcdf4 was not working, I had to isntall h5netcdf, I could not directly find it at the ECMW forum, but it could be worth trying it with an update of the netcdf4 converter.

---

I found out that encoder=netcdf4, cannot directly be used for byte arrays, therefore h5netcdf was used.

---

Ok. In my understanding `h5netcdf` can not be a full replacement for `netcdf4`. I'm fine with adding it as an dependency now. But leave a comment so we might wanna clean that up in future.

```suggestion
    "h5netcdf", # For new cds api. Maybe make optional / may not be needed with future netcdf4 updates
```